packages/backend/src/repositories/product.ts

```diff
@@ -224,4 +224,45 @@ export class ProductRepository {
       },
     });
   }
+
+  public async findSimilar(
+    city: string,
```

Comment:
Maybe remove city like option of similarity?

Reply:
it was in ticket requirements, but i can remove it if its necessary

Reply:
Ok, just interested

Reply:
@voonya, are you speaking about not sending "city" from FE and take it by "productId"?